src/libs/actions/Report.js

```diff
@@ -1837,6 +1842,7 @@ function flagComment(reportID, reportAction, severity) {
     const parameters = {
         severity,
         reportActionID,
+        isDevRequest,
```

Comment:
you could just use `isDevRequest:` and call `Environment.isDevelopment()` over here instead of passing it via params.

using `withEnvironment` in FlagCommentPage seems unnecessary

Reply:
oh i missed that function (weird, I swear I looked at Environment), that makes more sense